### DISCUSSION TITLE
无法登录

### DISCUSSION THREAD

**Discussion by evencyan** [iPanelDev/docs#42] [互助提问区]
登录时有如下报错：

连接时出现问题
400 Bad Request: "time"已过期

**Comment by evencyan** [comment]
环境：ubuntu，arm64，Microsoft.AspNetCore.App 6.0.32

**Reply by Zaitonn** [reply]
检查一下运行iPanel的服务器系统时间和登录所用设备的系统时间是否同步

一般情况下是时间不准导致的

**Reply by Zaitonn** [reply]
https://github.com/iPanelDev/iPanel/issues/14

**Reply by evencyan** [reply]
解决了，感谢
